deploy/etc/docker/manager.Dockerfile

```diff
@@ -9,20 +9,21 @@
 # base image
 FROM python:3.8.1-slim
 
-# install netcat
-RUN apt-get update && \
-    apt-get -y install netcat && \
-    apt-get clean
-
 # set working directory
-WORKDIR /usr/src/app
+WORKDIR /opt/zeta/manager
 
 # Add app
-COPY . /usr/src/app
+COPY build/manager /opt/zeta/manager
```

Comment:
I believe this should be 
`COPY ./src/mgmt/services/manager /opt/zeta/manager`
Since we don't actually build any binaries

Reply:
The build script takes care of copy over